test/test_checks.py

```diff
@@ -9,6 +21,29 @@ def test_get_noqa_lines(self):
         self.assertEqual(checker.get_noqa_lines(checker.get_file_contents()), [2])
 
 
+class TestFlake8Stdin(TestCase):
+    @expectedFailureIf(FLAKE8_VERSION > 2)
+    def test_stdin(self):
+        """Test using stdin."""
+        filepath = get_absolute_path('data/doubles.py')
+        env = os.environ.copy()
+        env['PYTHONIOENCODING'] = 'utf8'
+        with open(filepath, 'rb') as f:
+            # For some reason using "--select=Q" did suppress all outputs, so
+            # the result might contain non flake_quotes related errors
+            p = subprocess.Popen(['flake8', '-'], stdin=f, env=env,
+                                 stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            stdout, stderr = p.communicate()
+
+        stdout = stdout.decode('utf8').splitlines()
```

Comment:
This isn't `stdout` any more; this is `stdout_lines`. Can we please rename the new variable?

```python
stdout_lines = stdout.decode('utf8').splitlines()
```